apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/EarlySecurityChecks/ErrorDrawer.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { Button, Flex, IconsLegacy, Link } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+import DrawerFooter from "./DrawerFooter";
+import { withV3StyleProvider } from "~/renderer/styles/StyleProviderV3";
+import ErrorDisplay from "../../../ErrorDisplay";
+import { createCustomErrorClass } from "@ledgerhq/errors";
+import useEnv from "~/renderer/hooks/useEnv";
+import { useHistory } from "react-router";
+import { DeviceBlocker } from "../../../DeviceAction/DeviceBlocker";
+import { setDrawer } from "~/renderer/drawers/Provider";
+import TrackPage from "~/renderer/analytics/TrackPage";
+import { track } from "~/renderer/analytics/segment";
+
+export type Props = {
+  error: Error;
+  onClickRetry: () => void;
+  closeable?: boolean;
+};
+
+const NotFoundEntityError = createCustomErrorClass("NotFoundEntityError");
```

Comment:
It's really weird you need to create an error for this (i know it's to get the `ErrorDisplay` to work). But you are sure that in this case `error instanceof FirmwareNotRecognized` is not true ? The base class is `Error`, but you can still check if in the prototype chain (thx Javascript :pain:) you have this `FirmwareNotRecognized`